src/hooks/useQuizEvents/useQuizResetClick.ts

```diff
@@ -5,16 +5,23 @@ import {
   QuestionTypes,
   QuizAPIActionTypes,
 } from '../../components/CioQuiz/actions';
-import { QuizEventsReturn, QuizResultsResponse, QuizSharedResultsData } from '../../types';
+import {
+  IQuizProps,
+  QuizEventsReturn,
+  QuizResultsResponse,
+  QuizSharedResultsData,
+} from '../../types';
 import useQueryParams from '../useQueryParams';
 
 const useQuizResetClick = (
   resetQuizSessionStorageState: () => void,
   dispatchLocalState: React.Dispatch<ActionAnswerQuestion>,
   dispatchApiState: React.Dispatch<ActionQuizAPI>,
-  quizResults?: QuizResultsResponse | QuizSharedResultsData
+  quizResults?: QuizResultsResponse | QuizSharedResultsData,
+  quizBasePath?: IQuizProps['quizBasePath']
+  // eslint-disable-next-line max-params
```

Comment:
**suggestion**: I wonder if we should refactor this and pass an object as function parameter instead